docs/BreakingChanges/088 Unexpected InvalidCastException from InvokeMethod activity in WF4.md

```diff
@@ -0,0 +1,34 @@
+## 88: Unexpected InvalidCastException from InvokeMethod activity in WF4
```

Comment:
{
  "Id": "88",
  "Title": "Unexpected InvalidCastException from InvokeMethod activity in WF4",
  "Details": "Using a System.Activities.Statements.InvokeMethod that targets a method with a nullable parameter can throw an InvalidCastException.",
  "Suggestion": "This behavior was reverted in a .NET Framework 4.5 servicing release. Please update the .NET Framework 4.5 (or upgrade to .NET Framework 4.5.1 or later) to fix the issue.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/758736/unexpected-invalidcastexception-from-invokemethod-activity-in-wf4-after-net-4-5-install",
  "Markdown": null,
  "ApplicableApis": [
    "P:System.Activities.Statements.InvokeMethod.Parameters"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems#id=473476&triage=true&_a=edit",
  "Notes": "Activities can be defined in xaml (as well as programmatically), so it will be good to create a non-code analyzer to identify those. See the repro in the linked bug for an example.",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] P:System.Activities.Statements.InvokeMethod.Parameters